scripts/data-load.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+echo "register: $1"
+echo "phase: $2"
+
+echo ""
+echo "Downloading $2 field-records.json"
+rm -f field-records.json
+curl -sS https://field.$2.openregister.org/records.json > field-records.json
+
+echo ""
+[[ -e ../$1-data ]] || echo "cloning $1-data repo"
+[[ -e ../$1-data ]] || (cd .. && git clone git@github.com:openregister/$1-data.git)
+echo "../$1-data repo: checkout master && pull"
+cd ../$1-data && git checkout master && git fetch origin && git pull
+
+echo ""
+echo "Get credentials for $2 $1"
+[[ -e ../credentials ]] || echo "cloning credentials repo"
+[[ -e ../credentials ]] || (cd .. && git clone git@github.com:openregister/credentials.git)
+cd ../credentials && PASSWORD=`PASSWORD_STORE_DIR=~/.registers-pass pass $2/app/mint/$1`
+
+echo ""
+echo "Serialize tsv to $1.rsf"
+cd ../deployment
+if [[ -e ../$1-data/data/$2/$1/$1es.tsv ]]; then
```

Comment:
Plan is to change to a convention of using the single register name for these tsv files. Once done we can remove these plural checks.